tox.ini

```
@@ -1,9 +1,10 @@
 [tox]
-envlist = py38-{lilac,maple,nutmeg,olive,palm,quince},flake8,report
+envlist = py{38,311}-{lilac,maple,nutmeg,olive,palm,quince,redwood},flake8,report
 
 [gh-actions]
 python =
-    3.8: py38-lilac,py38-maple,py38-nutmeg,py38-olive,py38-palm,py38-quince,flake8
+    3.8: py38-lilac,py38-maple,py38-nutmeg,py38-olive,py38-palm,py38-quince,py38-redwood,flake8
+    3.11: py311-lilac,py311-maple,py311-nutmeg,py311-olive,py311-palm,py311-quince,py311-redwood,flake8
```

I'd argue that this ought to be sufficient:
```suggestion
    3.8: py38-lilac,py38-maple,py38-nutmeg,py38-olive,py38-palm,py38-quince,flake8
    3.11: py311-redwood,flake8
```